app/stats.py

```diff
@@ -75,8 +75,14 @@ def _is_app(app_id: str) -> bool:
 
 
 def get_popular(days: Optional[int]):
+    redis_key = f"popular:{days}"
```

Comment:
I'd rather use start and end dates in redis key, otherwise the cache would contain wrong data for the other day.

Reply:
Done. Landed some `black` fixes by accident but I'm going to stick it into CI soonish anyway.